firefox-ios/Client.xcodeproj/project.pbxproj

```diff
@@ -2349,20 +2351,6 @@
 			remoteGlobalIDString = F84B21BD1A090F8100AAB793;
 			remoteInfo = Client;
 		};
-		8C6FADF72EC1F3620022FB5C /* PBXContainerItemProxy */ = {
-			isa = PBXContainerItemProxy;
-			containerPortal = F84B21B61A090F8100AAB793 /* Project object */;
-			proxyType = 1;
-			remoteGlobalIDString = 43BE5774278BA4D900491291;
-			remoteInfo = RustMozillaAppServices;
-		};
-		8CB511232EC1F183004A2320 /* PBXContainerItemProxy */ = {
-			isa = PBXContainerItemProxy;
-			containerPortal = F84B21B61A090F8100AAB793 /* Project object */;
-			proxyType = 1;
-			remoteGlobalIDString = 2FCAE2191ABB51F800877008;
-			remoteInfo = Storage;
-		};
 		8CF609302EA8D46F00F069B6 /* PBXContainerItemProxy */ = {
```

Comment:
Not sure what's up here, I noticed the Localization script is also removing this in https://github.com/mozilla-mobile/firefox-ios/pull/30794